src/App.css

```diff
@@ -1,42 +1,48 @@
-#root {
-  max-width: 1280px;
-  margin: 0 auto;
-  padding: 2rem;
-  text-align: center;
+body {
+  background-color: #264653 !important;
 }
 
-.logo {
-  height: 6em;
-  padding: 1.5em;
-  will-change: filter;
-  transition: filter 300ms;
+* {
+  margin: 0;
+  padding: 0;
 }
-.logo:hover {
-  filter: drop-shadow(0 0 2em #646cffaa);
+
+/* Header */
+
+nav {
+  padding: 0 !important;
 }
-.logo.react:hover {
-  filter: drop-shadow(0 0 2em #61dafbaa);
+.header {
+  background-color: #2a9d8f !important;
 }
 
-@keyframes logo-spin {
-  from {
-    transform: rotate(0deg);
-  }
-  to {
-    transform: rotate(360deg);
-  }
+.navbar-brand,
+.nav-link {
+  color: white !important;
```

Comment:
Kolorki najlepiej utrzymywać w formie HEX i jako zdefiniowana zmienna gdzieś w osobnym pliku zmiennych CSS (nawet jesli to jest tylko kolor biały, to robimy zmienna white i jej uzywamy - tak to juz jest)